src/angular/ngVueLinker.js

```diff
@@ -30,7 +30,9 @@ export function ngVueLinker (componentName, jqElement, elAttributes, scope, $inj
     el: jqElement[0],
     data: reactiveData,
     render (h) {
-      return <Component {...{ directives }} {...{ props: reactiveData._v, on }} />
+      return <Component {...{ directives }} {...{ props: reactiveData._v, on }}>
+        {jqElement.html()}
+      </Component>
```

Comment:
Could you please enclose the `<Component></Component>` between parentheses?
```javascript
return (
  <Component {...{ directives }} {...{ props: reactiveData._v, on }}>
    {jqElement.html()}
  </Component>
)
```
so that opening and closing tag are aligned.